easybuild/easyblocks/n/numpy.py

```diff
@@ -66,7 +88,9 @@ def __init__(self, *args, **kwargs):
         self.sitecfg = None
         self.sitecfgfn = 'site.cfg'
         self.testinstall = True
-        self.testcmd = "cd .. && %(python)s -c 'import numpy; numpy.test(verbose=2)'"
+        # unset flags that can influence the testsuite
```

Comment:
Hmm, can we add some more details here, for example *how* are tests failing if `LDFLAGS` is not unset? @edmondac?

Reply:
See https://github.com/numpy/numpy/issues/2411

Reply:
I knew it was logged somewhere: https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/generic/fortranpythonpackage.py#L57